crates/orderbook/src/main.rs

```diff
@@ -273,6 +274,90 @@ struct Arguments {
     pub liquidity_order_owners: Vec<H160>,
 }
 
+impl std::fmt::Display for Arguments {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.shared)?;
+        writeln!(f, "bind_address: {}", self.bind_address)?;
```

Comment:
I think to not blow up `orderbook/src/main.rs` and `solver/src/main.rs` the `Arguments` struct and its `Display` implementation should probably be moved into their own files at this point.